### PR TITLE
[Document] add notebook instruction

### PR DESCRIPTION
**Pull request by hhbyyh**
## What changes were proposed in this pull request?

send the PR on behalf of Dave @dnielsen, update the AWS EC2 document.

## How was this patch tested?
manual review for the doc.

### CODE REVIEW

**Comment by hhbyyh** [comment]
cc @dnielsen, Please help review it.

If we're to merge it before a thorough review, please briefly state the business justification and confirm the notebook.sh has been tested by you or someone else.  Thanks.

**Comment by hhbyyh** [comment]
Also cc @shane-huang @helenlly since they were the original author for the page.

**Comment by dnielsen** [comment]
Thank you @hhbyyh. This section is required by the AWS Marketplace AMI team. They are waiting on us.

File: docs/docs/ProgrammingGuide/run-on-ec2.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 ---
+## **AWS Marketplace AMI**
 
+The easiest way to get started with BigDL is the BigDL AMI in the AWS Marketplace. It includes Apache Spark, BigDL,
+Jupyter Notebooks, Python and more. Within minutes you can create a Jupyter Notebook, upload some data and begin
+experimenting with Deep Learning.
```
Comment (jason-dai) [inline docs/docs/ProgrammingGuide/run-on-ec2.md]: why the line break here?
and change to "experimenting with Deep Learning using BigDL"

Comment (hhbyyh) [reply docs/docs/ProgrammingGuide/run-on-ec2.md]: Hi Jason, not sure what you mean by the line break...

Do you think we should append "Please note that it is highly recommended... " directly to this paragraph?
I break the first paragraph into 3 lines to make each line have the length around 100. In the generated doc, they will appear to be in the same paragraph.

I'll change it to "experimenting with Deep Learning using BigDL"

Comment (jason-dai) [reply docs/docs/ProgrammingGuide/run-on-ec2.md]: There is a line break between "Apache Spark, BigDL," and "Jupyter Notebooks, Python and more".

Don't break the line manually; the build tool will handle that.

Comment (hhbyyh) [reply docs/docs/ProgrammingGuide/run-on-ec2.md]: I see. I think the line break will not be visible in the generated doc. But I'll update the doc to be consistent with the project style.

**Comment by jason-dai** [comment]
LGTM

**Comment by dnielsen** [comment]
Great. Will this go live today?

**Comment by yiheng** [comment]
merge this change. Running a jenkins job to refresh the doc website

**Comment by yiheng** [comment]
The change is available now:
https://bigdl-project.github.io/master/#ProgrammingGuide/run-on-ec2/

**Comment by hhbyyh** [comment]
Thanks for updating the online doc.

**Comment by dnielsen** [comment]
Thank you all!

**Pull request merged**